src/zimscraperlib/download.py

```diff
@@ -65,8 +65,8 @@ def download(
 
 
 class YoutubeConfig(dict):
-    options = {}  # noqa: RUF012
-    defaults = {  # noqa: RUF012
+    options: ClassVar[Dict[str, Optional[Union[str, bool, int]]]] = {}
```

Comment:
We agreed on using the MappingProxy but I see that you eventually chose the `ClassVar`. Any reason to prefer it?

Reply:
Digging a bit more into the situation, it looks like `MappingProxyType` makes ruff happy but I'm pretty sure it doesn't really enhance the situation. You can't mutate the dict values anymore (but we don't do that) at the expense of using a different type and this makes debugging further pyright issues more difficult. 

With `ClassVar`, the type is clearly identified by the type checker and debugging further pyright issues was simple.